Lib/slapdtest/_slapdtest.py

```diff
@@ -330,6 +331,12 @@ def gen_config(self):
         }
         return self.slapd_conf_template % config_dict
 
+    def _include_directives(self):
```

Comment:
There is no need for an extra function. Instead of a new function, simply create a variable in the ``gen_config`` method and use it as ``'include_directives': include_directives,``.